aipal/frontend/src/template/kyselykerta/kyselykerta.html

```diff
@@ -56,7 +56,9 @@ <h3>{{'kyselykerta.vastausaika' | i18n}}</h3>
 <div ng-if="!uusi">
     <h2>{{'vastaajatunnus.lista' | i18n}} <ohje tunniste="kyselykerran_vastaajatunnukset"></ohje></h2>
     <button class="btn btn-primary e2e-luo-vastaajatunnuksia" ng-click="luoTunnuksiaDialogi()"
-            ng-if="muokkaustila && (!kysely.automatisoitu && kyselykerta.automaattinen === null) || kysely.tyyppi === 'tyoelamapalaute'" >
+            ng-if="muokkaustila &&
+            (!kysely.automatisoitu && kyselykerta.automaattinen === null) ||
```

Comment:
Oliko niin, että and arvioidaan ennen or:reja niin tuohon jälkimmäiseen ei huomioida muokkaustilaa. En ole varma onko tarkoituksella.